README.md

```diff
@@ -8,13 +8,13 @@
 
 # Azure Functions .NET Worker
 
-Welcome to the Azure Functions .NET Worker Repository. The .NET Worker provides .NET 5 support in Azure Functions, introducing an **Isolated Model**, running as an out-of-process language worker that is separate from the Azure Functions runtime. This allows you to have full control over your application's dependencies as well as other new features like a middleware pipeline.
+Welcome to the Azure Functions .NET Worker Repository. Since 2020, Azure Functions introduced an **Isolated Worker Model**, running as an isolated language worker that is separate from the Azure Function runtime. This allows you to have full control over your application's dependencies, easily incorporate Dependency Injection as well as other advanced .NET features such as middleware.
 
-A .NET Isolated function app works differently than a .NET Core 3.1 function app. For .NET Isolated, you build an executable that imports the .NET Isolated language worker as a NuGet package. Your app includes a [`Program.cs`](samples/FunctionApp/Program.cs) that starts the worker.
+A .NET Isolated worker process function works different to a .NET in-process function. For .NET Isolated, you build an executable that imports the .NET Isolated language worker as a NuGet package. Your app includes a [`Program.cs`](samples/FunctionApp/Program.cs) that starts the worker.
```

Comment:
```suggestion
A .NET Isolated worker process function works differently from a .NET in-process function. For .NET Isolated, you build an executable that imports the .NET Isolated language worker as a NuGet package. Your app includes a [`Program.cs`](samples/FunctionApp/Program.cs) that starts the worker.
```

Reply:
Although the specific preposition (to/from/than) seems to be a matter of region and preference; TIL! [Is it 'different from' or 'different than'? | Merriam Webster](https://www.merriam-webster.com/words-at-play/different-from-or-different-than)